build-system/routes/a4a-envelopes.js

```diff
@@ -35,6 +35,9 @@ app.use('/inabox/', (req, res) => {
     adUrl = urlPrefix.replace('localhost', 'ads.localhost') + adUrl;
   }
   adUrl = addQueryParam(adUrl, 'inabox', 1);
+  if (req.query.log) {
+    adUrl += '#log=' + req.query.log;
```

Comment:
Would this need url escaping?

Reply:
should be fine, we only allow `log={number}`.

Reply:
K should be safe